modules/vpc/variables.tf

```diff
@@ -44,6 +44,12 @@ variable "prefix" {
   type        = string
 }
 
+variable "internal_load_balancer_subnetwork_ip_cidr_range" {
+  default     = "10.2.0.0/23"
```

Comment:
can we add a bit to the description on where the default is coming from?

Reply:
Fixed by https://github.com/hashicorp/terraform-google-terraform-enterprise/pull/73/commits/1da4119d3fc6b7a231cb24747f6573edbaf33a79.